app/commands/recent.py

```diff
@@ -0,0 +1,50 @@
+
+from app.common.database.repositories import users, scores
+from app.common.constants import Mods
+from app.objects import Context
+from discord import Embed
+from discord import Color
+
+import config
+import app
+
+@app.session.commands.register(["recent", "last"])
+async def recent(context: Context):
+    """Displays your last play"""
+    if not (user := users.fetch_by_discord_id(context.message.author.id)):
+        await context.message.channel.send(content="You don't have an account linked!")
+        return
+
+    if not (score := scores.fetch_recent(user_id=user.id, mode=0, limit=1)):
+        await context.message.channel.send(content="No recent scores.")
+        return
+
+    score = score[0]
+    beatmapset = score.beatmap.beatmapset
+    rank = score.grade
+    max_combo = score.max_combo
+    accuracy = score.acc
+    n300 = score.n300
+    n100 = score.n100
+    n50 = score.n50
+    nmiss = score.nMiss
+    pp = score.pp
+    nscore = score.total_score
+    mods = Mods(score.mods).short
+
+    embed = Embed(
+        title=f"{beatmapset.title} +{mods}",
```

Comment:
Should we do `beatmapset.full_name` here, or does that look bad?

Reply:
we should

Reply:
ok